src/validators/validateDependencies.ts

```diff
@@ -8,21 +8,21 @@ const isValidVersionRange = (v: string): boolean => {
 		v == "*" ||
 		v === "" ||
 		v === "latest" ||
-		(v.indexOf && v.indexOf("git") === 0) ||
+		(typeof v === "string" && v.startsWith("git")) ||
 		// https://pnpm.io/next/workspaces#workspace-protocol-workspace
 		/^workspace:((\^|~)?[0-9.x]*|(<=?|>=?)?[0-9.x][\-.+\w]+|\*)?$/.test(v) ||
 		// https://pnpm.io/next/catalogs
-		(v.indexOf && v.indexOf("catalog:") === 0) ||
-		(v.indexOf && v.indexOf("npm:") === 0) ||
+		(typeof v === "string" && v.startsWith("catalog:")) ||
+		(typeof v === "string" && v.startsWith("npm:")) ||
```

Comment:
Explaining: `v.indexOf` is always true in strings, but `v` is not always a `string`. Its type annotation is wrong. It might be an object. This is an existing bug. Rather than fix up types everywhere, I just switched this to a direct `typeof` check.